src/assets/scss/application.scss

```diff
@@ -290,4 +290,38 @@ $nav-active-border-thickness: 5px;
   display: block;
   margin: govuk-spacing(2) 0;
   @include govuk-font($size: 48, $weight: bold);
-}
\ No newline at end of file
+}
+
+.language-select {
+  float: right;
```

Comment:
I would probably choose to avoid using float here (alternatively, clear the float) because it is causing some layout issues on smaller screens where it's almost going in line with the heading element instead of staying above it.

Reply:
I've added the clear but left the float just because we need the bottom-border to be the same size of the navigation otherwise it will span the whole container width.
I have tested it on different screen resolutions and it looks fine.